webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -333,13 +331,7 @@ const JobQueue: React.FC<Props> = ({ bodyNoPadding, selectedRp, jobState }) => {
   }, [selectedRp]);
 
   return (
-    <Page
-      bodyNoPadding={bodyNoPadding}
-      className={css.base}
-      containerRef={pageRef}
-      headerComponent={<div />}
-      id="jobs"
-      title="Job Queue by Resource Pool">
+    <div className={css.base} id="jobs" title="Job Queue by Resource Pool">
```

Comment:
didn't the title prop on `Page` set the title of the document?

Reply:
Yeah, you're right. I'll add a condition to `ResourcepoolDetail` that should fix this.